MzmlReader/MzmlReader.cs

```diff
@@ -329,9 +329,23 @@ private static void ParseBase64Data(ScanAndTempProperties scan, Run run, bool ex
                     //Check to see if we have a basepeak we can add points to
                     else
                     {
+                        bool withinRTtolerance = false;
                         foreach (BasePeak bp in run.BasePeaks.Where(x => Math.Abs(x.RetentionTime - scan.Scan.ScanStartTime) <= rtTolerance && Math.Abs(x.Mz - scan.Scan.BasePeakMz) <= massTolerance))
                         {
                             bp.Spectrum.Add(spectrum.Where(x => Math.Abs(x.Mz - bp.Mz) <= massTolerance).OrderByDescending(x => x.Intensity).First());
+                            withinRTtolerance = true;
+                        }
+                       if (withinRTtolerance == false)
+                        {
+                            spectrum.Select(x => Math.Abs(x.Mz - scan.Scan.BasePeakMz) <= massTolerance);

```

Comment:
This isn't an in-place method I don't think, so the evaluated result doesn't seem to go anywhere? I think we can delete this line?